src/examples/fake-devicegraphs-yaml-format.md

```diff
@@ -0,0 +1,304 @@
+# YAML File Format For Fake Device Graphs
```

Comment:
With this repo restructuring, we might want to move this document to a toplevel doc/ directory.